content/rooms/fragments/v2-state-res.md

```diff
@@ -90,11 +90,13 @@ event in its `auth_events`.
 The *mainline of P*<sub>0</sub> is the list of events
     [*P*<sub>n</sub> , ... , *P*<sub>1</sub>, *P*<sub>0</sub>].
 
-Given another event *e*, the *closest mainline event to*
-*e* is the first event encountered in the mainline when iteratively
-descending through the `m.room.power_levels` events in the `auth_events`
-starting at *e*. If no mainline event is encountered when iteratively
-descending through the `m.room.power_levels` events, then the closest
+Given another event *e* = *e<sub>0</sub>* we can compute a similar list of
```

Comment:
The old definition didn't seem to allow for `e` (`e_0`) itself to be the *closes mainline event to e*, but the one seems to allow it. Is this correct?

Also, is `e` meant to be an arbitrary event here or another `m.room.power_levels` event?

Reply:
Not sure, I'll consider this more carefully.

Reply:
> The old definition didn't seem to allow for `e` (`e_0`) itself to be the _closest mainline event to e_, but the new one seems to allow it. Is this correct?

I think there is some leeway here. What matters is that part (1) of the mainline ordering is unchanged (we don't use this notion of closest mainline event anywhere else AFAICS).

> Also, is `e` meant to be an arbitrary event here or another `m.room.power_levels` event?

I think `e` is arbitrary including the possibility that `e` is another `m.room.power_levels` event.